apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.BenchmarkingTool/Program.cs

```diff
@@ -0,0 +1,244 @@
+
+namespace Google.Cloud.Storage.V1.BenchmarkingTool
+{
+    public static class Program
+    {
+        private static readonly string s_projectId = TestEnvironment.GetTestProjectId(TestEnvironment.TestProjectEnvironmentVariable);
+
+        private static readonly string[] s_logHeaders = new string[] { "Op", "ObjectSize", "AppBufferSize", "LibBufferSize", "Crc32cEnabled", "MD5Enabled", "ApiName", "CpuTimeUs", "ElapsedTimeUs", "Status" };
+
+        private static Stopwatch s_stopwatch;
+
+        private const long TicksPerMicrosecond = 10;
+
+        private static void Main(string[] args)
+        {
+            try
+            {
+                var configuration = Configuration.FromCommandLineArguments(args);
+
+                var logger = new PerformanceLogger(configuration.OutputFolder);
+                logger.Log(s_logHeaders);
+
+                var client = StorageClient.Create();
+
+                s_stopwatch = Stopwatch.StartNew();
+
+                for (int i = 1; i <= configuration.NumOfSamples; ++i)
+                {
+                    Console.WriteLine($"Running W1R3 iteration number {i}..");
+                    RunW1R3Benchmark(configuration, client, logger);
+                }
+
+                s_stopwatch.Stop();
+            }
+            catch (ArgumentException e)
+            {
+                var originalColour = Console.ForegroundColor;
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.Write(e.Message);
+                Console.ForegroundColor = originalColour;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+            }
+        }
+
+        private static void RunW1R3Benchmark(Configuration configuration, StorageClient client, PerformanceLogger logger)
+        {
+            try
+            {
+                CreateBucket(configuration.Bucket, client);
+
+                Write(configuration, client, logger, "Write");
+                Read(configuration, client, logger, "Read[1]");
+                Read(configuration, client, logger, "Read[2]");
+                Read(configuration, client, logger, "Read[3]");
+            }
+            finally
+            {
+                DeleteBucket(client, configuration.Bucket);
+            }
+        }
+
+        private static Bucket CreateBucket(string bucketName, StorageClient client)
+        {
+            var bucket = client.CreateBucket(s_projectId, new Bucket { Name = bucketName });
+            SleepAfterBucketCreateDelete();
+            return bucket;
+        }
+
+        private static void DeleteBucket(StorageClient client, string bucketName)
+        {
+            try
+            {
+                client.DeleteBucket(bucketName, new DeleteBucketOptions { UserProject = null, DeleteObjects = true });
+            }
+            catch (GoogleApiException)
+            {
+                //Some tests fail to delete buckets due to object retention locks etc.
```

Comment:
Nit: add a space after `//` here and below.
(If we don't have a space in the test fixture code this was copied from, feel free to update that in the same PR...)